pages/knowledge/colours-and-contrast/graphical-objects/README.md

```diff
@@ -0,0 +1,50 @@
+---
+navigation_title: "Colour contrast for graphical objects"
+position: 5
+changed: "2019-04-21"
+---
+
+# Colour contrast for graphical objects
+
+**The term “graphical object” typically applies to stand-alone icons and to information graphics. The visual complexity of such objects often requires some differentiation with regards to contrast. The focus lies on the parts of the graphical object required to understand the content.**
+
+## Minimal contrast ratio
+
+Version 2.1 of the [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast) of `3:1` against adjacent colour(s) for the parts, which are required to understand the meaning or content of a graphical object.
```

Comment:
I would use "minimal contrast ratio of 3:1 for all User Interface components" as link text.

Reply:
We should also point out that we are referring to AA, not AAA.

Reply:
**To Do:** Correct link text

Reply:
done